Basic FE project template/ui-src/styles/style.scss

```diff
@@ -1,2 +1,4 @@
-@import 'general/index';
+@import "general/index";
 @import "components/index";
+@import "components/why-beleasing/why-beleasing";
+@import "queries.scss";
```

Comment:
this needs to be removed, we do not need a file named queries, each component will have it's own queries